cub/cub/device/dispatch/dispatch_transform.cuh

```diff
@@ -346,15 +343,24 @@ struct dispatch_t<StableAddress,
   CUB_RUNTIME_FUNCTION _CCCL_FORCEINLINE cudaError_t Invoke(ActivePolicyT active_policy = {})
   {
     auto wrapped_policy = detail::transform::MakeTransformPolicyWrapper(active_policy);
-#ifdef _CUB_HAS_TRANSFORM_UBLKCP
+    const auto seq      = ::cuda::std::index_sequence_for<RandomAccessIteratorsIn...>{};
     if constexpr (Algorithm::ublkcp == wrapped_policy.GetAlgorithm())
     {
-      return invoke_ublkcp_algorithm<ActivePolicyT>(::cuda::std::index_sequence_for<RandomAccessIteratorsIn...>{});
+      return invoke_async_algorithm<ActivePolicyT>(
+        bulk_copy_align,
+        [&](int tile_size) {
+          return bulk_copy_smem_for_tile_size<RandomAccessIteratorsIn...>(tile_size, bulk_copy_align);
+        },
```

Comment:
The only thing this lambda captures is the static `bulk_copy_align`

Could we rather turn this into a functor with a call operator?

Reply:
Discussed offline and changed to capture `[this]` instead.